src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v1/TransactionExpirationQueueConsumer.kt

```diff
@@ -139,10 +139,30 @@ class TransactionExpirationQueueConsumer(
           }
         }
         .filter {
+          val refundableCheckRequired = isRefundableCheckRequired(it)
           val refundable = isTransactionRefundable(it)
+          val refundableWithoutCheck = refundable && !refundableCheckRequired
           logger.info(
-            "Transaction ${it.transactionId.value()} in status ${it.status}, refundable: $refundable")
-          refundable
+            "Transaction ${it.transactionId.value()} in status ${it.status}, refundable : $refundable, without check : $refundableWithoutCheck")
+          if (refundableCheckRequired) {
+            val tracingInfo = queueEvent.second
+            val binaryData =
+              if (tracingInfo == null) {
+                BinaryData.fromObject(event)
+              } else {
+                BinaryData.fromObject(QueueEvent(event, tracingInfo))
+              }
+            deadLetterTracedQueueAsyncClient
+              .sendAndTraceDeadLetterQueueEvent(
+                binaryData,
+                DeadLetterTracedQueueAsyncClient.ErrorContext(
+                  transactionId = TransactionId(event.transactionId),
+                  transactionEventCode = event.eventCode,
+                  errorCategory = DeadLetterTracedQueueAsyncClient.ErrorCategory.PROCESSING_ERROR),
```

Comment:
Can we think about a new error category for this kind of dead letter event? Something more specialized then `PROCESSING_ERROR`. This way those errors will be searchable into dead letter dashboard. 
What about UNPROCESSABLE_REFUND or something like that?

Reply:
`REFUND_MANUAL_CHECK_NEEDED`  sounds good

Reply:
renamed in `REFUND_MANUAL_CHECK_REQUIRED`